openmetadata-ui/src/main/resources/ui/src/components/DatasetDetails/DatasetDetails.component.tsx

```diff
@@ -730,7 +730,7 @@ const DatasetDetails: React.FC<DatasetDetailsProps> = ({
                     />
                   </Col>
                   <Col offset={1} span={6}>
-                    <div className="border-1 border-main rounded-6">
+                    <div className="global-border rounded-4">
```

Comment:
We can use Card component instead adding custom css via class

Reply:
Actually, we can't as card components as many other properties too, that we don't need, and for eliminating them we eventually need to write more custom  CSS